be/src/olap/options.h

```diff
@@ -26,14 +26,25 @@
 namespace doris {
 
 struct StorePath {
-    StorePath() : capacity_bytes(-1) { }
-    StorePath(const std::string& path_, int64_t capacity_bytes_)
-        : path(path_), capacity_bytes(capacity_bytes_) { }
+    StorePath() : capacity_bytes(-1), storage_medium(TStorageMedium::HDD) { }
+    StorePath(const std::string &path_, int64_t capacity_bytes_)
```

Comment:
```suggestion
    StorePath(const std::string& path_, int64_t capacity_bytes_)
```
Doris prefer `const std::string& path` to `const std::string &path`

Reply:
So Doris align pointer to the right, and reference to the left? I think that's not a good idea.
In clang-format, pointer and reference alignment style is a single option, "PointerAlignment".
http://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
Doris align both reference and pointer to the left.

Reply:
ok, I chanced to see this, mistakely think that Doris align pointer to the right   
https://github.com/apache/incubator-doris/blob/42a4fff562090fc40bcab48684b4a61614f8d8f8/be/src/tools/meta_tool.cpp#L105